test/persistence-hooks.suite.js

```diff
@@ -1,7 +1,10 @@
 var ValidationError = require('../').ValidationError;
 var traverse = require('traverse');
 
-module.exports = function(dataSource, should) {
+module.exports = function(dataSource, should, connectorCapabilities) {
+  if (!connectorCapabilities || connectorCapabilities.replaceOrCreateReportsNewInstance === undefined) {
```

Comment:
I think it will be better to "normalize" `connectorCapabilities` so that we don't have to check in all places whether it's defined.


```js
module.exports = function(dataSource, should, connectorCapabilities) {
  if (!connectorCapabilities) connectorCapabilities = {};
  // ...
}
```

Thoughts?